app/views/sprint-5/monitor/cases.html

```diff
@@ -0,0 +1,81 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Manage interventions and services
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+  {% include "./includes/primary-navigation.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column">
+      <h1 class="govuk-heading-xl">
+        My cases
+      </h1>
+    </div>
+    <div class="group-by">
+      <h2 class="govuk-heading-m group-by-heading">
+        Group by:
+      </h2>
+      <ul class="group-by-links">
+        <li class="group-by-link govuk-tag">
+          <a aria-current="page" href="cases">Details</a>
+        </li>
+        <li class="group-by-link">
+          <a href="stages">Stages</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+
+  <table class="govuk-table">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header" aria-sort="ascending">Referral</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Service User</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Intervention category</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Provider</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Caseworker</th>
+        <th scope="col" class="govuk-table__header" aria-sort="none">Action</th>
+      </tr>
+    </thead>
+
+    <tbody class="govuk-table__body">
+      {% for referral in referrals %}
+        {% set referralIndex = loop.index0 %}
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell" rowspan="{{referral.interventions.length}}">
+            {{ referral.reference }}
+          </td>
+          <td class="govuk-table__cell" rowspan="{{referral.interventions.length}}">
+            {{ referral.serviceUser.name }}
+          </td>
+
+          {% for intervention in referral.interventions %}
+            <td class="govuk-table__cell">
+              {{ intervention.name | capitalize }}
+            </td>
+            <td class="govuk-table__cell">
+              {{ intervention.providerName }}
+            </td>
+            <td class="govuk-table__cell">
+              {{ intervention.assignedCaseworker }}
+            </td>
+            <td class="govuk-table__cell">
+              <a class="govuk-link" href="cases/{{caseIndex}}/interventions/{{intervention.id}}">View</a>
+            </td>
+
+            {% if referral.interventions.length > 1 and not loop.last %}
```

Comment:
I'm not quite sure what this condition is intending to do, but it seems to output unbalanced HTML:

```html
        <tr class="govuk-table__row">
          <td class="govuk-table__cell" rowspan="2">
            NR0002
          </td>
          <td class="govuk-table__cell" rowspan="2">
            Bob Brown
          </td>

          
            <td class="govuk-table__cell">
              Accommodation
            </td>
            <td class="govuk-table__cell">
              XYZ Provider Name
            </td>
            <td class="govuk-table__cell">
              Liam Johnson
            </td>
            <td class="govuk-table__cell">
              <a class="govuk-link" href="cases//interventions/0">View</a>
            </td>

            
            </tr>
            

          
            <td class="govuk-table__cell">
              Social inclusion
            </td>
            <td class="govuk-table__cell">
              XYZ Provider Name
            </td>
            <td class="govuk-table__cell">
              Rebecca Green
            </td>
            <td class="govuk-table__cell">
              <a class="govuk-link" href="cases//interventions/1">View</a>
            </td>

            

          
        </tr>
```

I think you're missing an opening `<tr>`. I'm going to approve and merge because it looks okay in a browser and I'd like these changes so that I can fix #43, but might be worth taking a look at.